package.json

```diff
@@ -14,12 +14,12 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.7",
-    "@docusaurus/preset-classic": "2.0.0-beta.7",
-    "@mdx-js/react": "1.6.22",
-    "clsx": "1.1.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@docusaurus/core": "^2.0.0-beta.9",
+    "@docusaurus/preset-classic": "^2.0.0-beta.9",
+    "@mdx-js/react": "^1.6.21",
+    "clsx": "^1.1.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
```

Comment:
Why did we downgrade?

Reply:
That's something pulled out by yarn dependencies. I've made a commit with another yarn package update again and seems like this is not changed anyway.

Reply:
I'd rather not have any range version dependencies.

Reply:
These are not a problem anymore.